documentation/book/assembly-upgrade.adoc

```diff
@@ -3,20 +3,83 @@
 // master.adoc
 
 [id='assembly-upgrade-{context}']
-= {ProductName} and Kafka upgrades
+= Upgrade
```

Comment:
I think the title should make it more clear what upgrades are we talking about ... I think the original title was better.

Reply:
I was going with the other highest level titles like 'Security'. No strong feelings, so have reverted.